controllers/nodenetworkconfigurationpolicy_utils.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The Kubernetes NMState Authors.
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"regexp"
+	"strings"
+)
+
+// The function strips disarranged parts of the error messages, so it is easier to read for the user
+func formatErrorString(err error) string {
+	var sb strings.Builder
+	errorMessage := err.Error()
+	errorLines := strings.Split(errorMessage, "\n")
+	index := 0
+
+	re := regexp.MustCompile(`\s*(.*--no-commit --timeout \d+: 'exit status \d+') '{2}\s'`)
```

Comment:
Stuff like this is very fragile, imaging that we change the arguments order and we do --timeout and then --no-commit